docs/spelling-wordlist.txt

```diff
@@ -23,6 +23,7 @@ shortnames
 subcommand
 subcommands
 validator
+config
```

Comment:
This was failing in `docs(spelling)` checks because of this line

`Add this to ~/.config/fish/completions/bowtie.fish:`